lib/pure/collections/sequtils.nim

```diff
@@ -704,6 +704,31 @@ template newSeqWith*(len: int, init: untyped): untyped =
     result[i] = init
   result
 
+when not defined(nimscript):
+  import macros
+  macro asArray*(values: typed, targetType: typedesc): untyped =
+    ## converts a static array to an array of type ``targetType`` by converting
```

Comment:
I think you should change this to something like this:

> applies a type conversion to each of the elements in the specified array literal. Each element is converted to the ``targetType`` type.

Reply:
@dom96 Error from nim while executing `nim e tests\test_nimscript.nims`
> lib\core\macros.nim(735, 1) Error: cannot generate VM code for macro dumpTree(s: untyped): untyped